nexus/types/src/deployment.rs

```diff
@@ -488,6 +505,30 @@ impl<'a> BlueprintDisplay<'a> {
             ],
         )
     }
+
+    // Return tables representing a [`ClickhouseClusterConfig`] in a given blueprint
+    fn make_clickhouse_cluster_config_tables(
+        &self,
+    ) -> Option<(KvListWithHeading, BpSledSubtable, BpSledSubtable)> {
+        let Some(config) = &self.blueprint.clickhouse_cluster_config else {
+            return None;
+        };
+
+        let diff_table =
+            ClickhouseClusterConfigDiffTables::single_blueprint_table(
+                BpDiffState::Unchanged,
+                config,
+            );
+
+        Some((
+            diff_table.metadata,
+            diff_table.keepers,
+            // Safety: The call to
+            // `ClickhouseClusterConfigDiffTables::single_blueprint_table`
+            // always returns all tables.
```

Comment:
I haven't looked at `ClickhouseClusterConfigDiffTables` yet, but leaving a note mostly for myself for when I get to it: could it return a different type so we don't have to unwrap here?